source/code/plugins/patch_management_lib.rb

```diff
@@ -11,9 +12,13 @@ class LinuxUpdates
     @@prev_hash = ""
     @@delimiter = "_"
     @@force_send_last_upload = Time.now
+
     MAJOR_MINOR_VERSION_REGEX = /([^\.]+)\.([^\.]+).*/
     OMS_ADMIN_FILE = "/etc/opt/microsoft/omsagent/conf/omsadmin.conf"
     SCX_RELEASE_FILE = "/etc/opt/microsoft/scx/conf/scx-release"
+    CURRENT_UPDATE_RUN_FILE = "/tmp/schedule_run.id"
+    SCHEDULE_NAME_VARIABLE = "SCHEDULE_NAME"
+    APT_GET_START_DATE_TIME_FORMAT = "%Y-%m-%d %H:%M:%S"
 
     def self.getAgentDetails()
         ret = {}
```

Comment:
you can use the OMS::Configuration class to get all the settings. require_relative 'oms_configuration'